release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -655,6 +655,13 @@ The Machine Config Operator (MCO) no longer automatically reboots all correspond
 
 For more information, see xref:../architecture/control-plane.adoc#understanding-machine-config-operator_control-plane[Understanding the Machine Config Operator].
 
+[id="ocp-4-7-bmh-soft-shutdown"]
+==== BareMetalHost API supports soft shutdown
+
+In {product-title} 4.6, when the online flag in the BareMetalHost API is set to `false`, the Bare Metal Operator shuts down nodes "hard." That is, it turns the power off without giving the operating system or workloads time to react. In {product-title} 4.7 and subsequent releases, the API sends the node's operating system a signal telling it to shut down, and then waits for the node to power off in "soft" mode. If the operating system does not shut down the node within three minutes, the Bare Metal Operator executes a "hard" shutdown.
+
+{product-title} 4.8 will execute a "hard" shutdown for remediation purposes, such as if there is a known problem with the node. The behavior of executing a "hard" shutdown for remediation purposes will be back ported to {product-title} 4.7.
```

Comment:
Hi @johnwilkins  - we should avoid predicting what will happen in a future release. Instead of saying {product-title} 4.8, please say something like: "In a future release of {product-title}". Same for the back porting.